Makefile

```diff
@@ -88,8 +88,11 @@ integration-in-ci:
 	make create-tmp-env-ci && \
 	TMP_ENV_CI=tmp.env.ci docker compose -f docker-compose.yml up integration-tests
 
+# NODE_OPTIONS line should be removed whenever we upgrade yarn.lock to see if
+# heap allocation issue has been resolved in dependencies (fails at 2048).
 execute-integration-from-within-container:
 	yarn install && \
+	NODE_OPTIONS="--max-old-space-size=3072" \
```

Comment:
@bodymindarts @krtk6160 could you please confirm if we have enough memory assigned in execution environment/pod ?

Reply:
I think we should have enough memory